common/src/main/java/com/revenuecat/purchases/common/OfferingFactories.kt

```diff
@@ -54,15 +54,16 @@ fun JSONObject.createPackage(
     offeringIdentifier: String
 ): Package? {
     val packageIdentifier = getString("identifier")
-    val productGroupIdentifier = optString("platform_product_group_identifier")
     // TODO handle INAPP products
```

Comment:
We should start thinking about these cases as well. But I think it's ok for this PR